server/model/traces.go

```diff
@@ -105,8 +105,8 @@ func spanType(attrs Attributes) string {
 }
 
 func spanDuration(span Span) string {
-	timeDifference := span.EndTime.Sub(span.StartTime)
-	return fmt.Sprintf("%d", int64(timeDifference))
+	timeDifference := timeDiff(span.StartTime, span.EndTime)
+	return fmt.Sprintf("%d", durationInNanoseconds(timeDifference))
```

Comment:
is this `durationInNanoseconds` needed? you need to follow an extra indirection that is only a single line.

Reply:
Actually, you don't even need to cast to int, `%d` can handle int64 correctly: https://go.dev/play/p/8Vth4PCiQyO

```suggestion
	return fmt.Sprintf("%d", timeDifference.Nanoseconds())
```

Reply:
Shoot sorry, I didn't see this. I'll add them in my next PR